ports/psoc6/boards/CY8CKIT-062S2-AI/manifest.py

```diff
@@ -0,0 +1,4 @@
+freeze("$(PORT_DIR)/freeze")
+include("$(MPY_DIR)/extmod/asyncio")
+require("mip")
+require("neopixel")
```

Comment:
Do we?

Reply:
The freeze we still need for the lfs files or?
The neopixel could be removed it for all boards.
Also my understanding is that mip is by default included. But we have to check that, and specially make mip work again 😄 
Let´s handle it in a separate PR.

Reply:
ok..so far yes we still need it for fs.
We can take it in another PR. All good from my side